### ISSUE TITLE
Feature Request - Markdown to Discord

### ISSUE THREAD

**Issue by ReenigneArcher** [facelessuser/pymdown-extensions#1389]
This looks like a cool project. I came across it while searching for a solution to a problem I'm having. I have a discord bot that returns the contents of github wiki pages (markdown) in an embed of a discord message. The issue is some parts of markdown are not supported in discord, but some are.

Would you be interested in adding an extension that can convert markdown to discord flavored markdown?

Either way, cool project!

**Comment by facelessuser** [comment]
Ultimately, this project was a number of extensions that, for one reason or another, I had a specific need for. I then made them available publicly. The extension pack is not meant to satisfy everyone's needs, but mainly those that align with similar needs that I have/had. People often cherrypick the extensions that are most useful to them and forego using the ones that do not.

While I do occasionally extend a specific extension with features I may not specifically use, in order to make the existing extension more useful to the general population of users, I don't usually add new extensions that I personally have no need for.

Basically, for every extension I add, I increase the maintenance burden. As time goes on, managing that burden has become more important, especially since this is not the only project I manage or contribute to.

While I can see how this would be useful for your specific use case, it isn't that useful to the general population, and it doesn't satisfy a personal need for me at this time either.

I appreciate your interest in this project, but as things currently stand, such a request is simply more than I am willing to take on in this project. I hope this answer makes sense 🙂.
